app/assets/stylesheets/application.scss

```diff
@@ -8,15 +8,29 @@ $button-back: #FFC107;
   min-height: 70vh;
 }
 
-body, .body-back {
+body {
   background-color: #B6B6B6;
 }
 
+.small {
+  font-size: 8px;
+  color: #ffffff;
+}
+
+.body-back {
+  background-color: #ffffff;
```

Comment:
Color literals like `#ffffff` should only be used in variable declarations; they should be referred to via variable everywhere else.